addon/utils/get-script.js

```diff
@@ -0,0 +1,30 @@
+export default function getScript(source, callback) {
+  /**
+   * Loads and executes a script asynchronously; using this here so the
+   * reCaptcha script isn't executed on every page, but only when the user
+   * hits the sign up page.
+   *
+   * getScript snippet taken from https://stackoverflow.com/a/28002292
+   */
+  var script = document.createElement("script");
+  var prior = document.getElementsByTagName("script")[0];
+  script.async = 1;
+
+  script.onload = script.onreadystatechange = function(_, isAbort) {
+    if (
+      isAbort ||
```

Comment:
what is this `isAbort` argument? I can't seem to get a value for the second argument in these event handlers.

Reply:
Full disclosure, I did not write this - this is a jQuery-free getScript alternative from https://stackoverflow.com/a/28002292, and I basically dug into this code about the same amount I would have if I'd done `npm i get-script`.

I did a quick search and I can't find any documentation that the second argument to the callback is supposed to be an error/abort message, but I'm reticent to remove it in case there are circumstances where that is what prevents it from entering an infinite loop.